src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs

```diff
@@ -21,11 +22,29 @@ class ConsoleShell
         /// <returns>An integer value which should be used as exit code for the process.</returns>
         public static int Start(string bannerText, string helpText, string[] args)
         {
+            return Start(InitialSessionState.CreateDefault2(), bannerText, helpText, args);
+        }
```

Comment:
This could be written with the `=>` syntax and dropping the braces, but I guess that's a matter of one's preference. 🙂 

```csharp
public static int Start(string bannerText, string helpText, string[] args) 
    => Start(InitialSessionState.CreateDefault2(), bannerText, helpText, args);
```